README.md

```diff
@@ -25,129 +25,67 @@ Setup of tools that allow you to play with & test out the UI with ease.
 
-python -m pip install pyserial
+### 2. Install an IDE
 
-Watch some text scroll before your eyes. Bam. You're sorted. 
+You will need to install an IDE. This can be anything you like but we'll use PyCharm in this guide.
 
-### 7. Tell Eclipse where to find your Python interpreter
+Head to: https://www.jetbrains.com/pycharm/download/#section=windows
 
-Open Eclipse (if you closed it). Go to the toolbar, and select Window > Preferences. Then, on the side-menu in Preferences, go to PyDev > Interpreters > Python. 
+Download the "Community version" as it is the free version and follow the installer steps.
 
-Click on “Browse for python/pypy exe”, and find your python.exe file, in the folder you installed it.
 
-All of Kivy's libraries should automatically be loaded into Eclipse as well. 
+### 3. Get easycut-smartbench from GitHub into your IDE 
 
-Click "Apply and Close". 
+This step will vary depending on your IDE so if you use something other than PyCharm, consult your IDE's website.
 
-Awesome. Almost done. 
+Open PyCharm, go to `File > Settings > Version Control > GitHub` and connect your account. For more details click [here](https://www.jetbrains.com/help/pycharm/github.html)
 
-### 8. Get easycut-smartbench from GitHub into Eclipse 
+Then, go to `Git > Clone` and copy and paste the link to the GitHub repository in the URL section and select where you'd like the repository to get stored **locally**. Make sure you remember this location. For more details click [here](https://www.jetbrains.com/help/pycharm/manage-projects-hosted-on-github.html)
 
-In Eclipse, go to the toolbar and select Window > Show View > Other. 
+If the code shows a lot of errors, don't worry, it's expected at this stage.
 
-In the pop-up, expand the "Git" folder, and click on "Git Repositories". Click OK. 
+### 4. Select your interpreter in your IDE
 
-Then, in the Eclipse toolbar go to File > Import. 
+Once again, this step will vary depending on your IDE, so please consult your IDE's website if you use something other than PyCharm.
 
-In the pop-up, select Projects from Git, and then Next. 
+In PyCharm, when you cloned the repository, you should have gotten a notification asking you to select the interpreter. If you did, click on it and select `Python 2.7` from the dropdown list.
 
-Click Clone from URI. 
+If you didn't get the notification, navigate to `File > Settings > Project: <project name> > Python Interpreter` and select `Python 2.7` as your interpreter.
 
-Paste the following into the URI box: 
+If `Python 2.7` does not show up in the list, go to `Add Interpreter > Add Local Interpreter` and navigate to the location where you installed python and select the `python.exe` file. If you installed in the default location, this should be `C:/Python27/python.exe`.
 
-https://github.com/YetiTool/easycut-smartbench.git
+For additional information, click [here](https://www.jetbrains.com/help/pycharm/configuring-python-interpreter.html#add-existing-interpreter)
```

Comment:
Nice to see that you've covered all cases of what might happen here - really helpful! 😁